src/containers/forms/RedeemPointsForm.jsx

```diff
@@ -88,7 +88,7 @@ class RedeemPointsForm extends Component {
     const { type } = this.props.message;
     if (type !== prevProps.message.type) {
       if (type === 'success') {
-        setTimeout(() => { this.handleCloseModal(); }, 2000);
+        setTimeout(() => { this.handleCloseModal(); }, 3000);
```

Comment:
Have the 3000 as a constant somewhere in the utilities section or constants section. Easier for config that way.